src/streams/mod.rs

```diff
@@ -334,28 +342,30 @@ impl Streams {
     }
 
     /// Lookup the "main" SSRC and mid for a given SSRC(main or RTX).
-    pub(crate) fn mid_ssrc_rx_by_ssrc_or_rtx(&mut self, ssrc: Ssrc) -> Option<(Mid, Ssrc)> {
+    pub(crate) fn mid_ssrc_rx_by_ssrc_or_rtx(
+        &mut self,
+        now: Instant,
+        ssrc: Ssrc,
+    ) -> Option<(Mid, Ssrc)> {
         // A direct hit on SSRC is to prefer. The idea is that mid/rid are only sent
         // for the initial x seconds and then we start using SSRC only instead.
-        if let Some(r) = self.source_keys_rx.get(&ssrc).copied() {
-            return Some(r);
+        if let Some(r) = self.rx_lookup.get_mut(&ssrc) {
+            r.last_used = now;
+            return Some((r.mid, r.main));
         }
 
-        // The receiver/source SSRC might already exist.
```

Comment:
Is this not still the case, at least for non-simulcast tracks?

Reply:
Hm. Yes. Why did I drop this doc?